packages/framework/esm-styleguide/src/datepicker/openmrs/openmrs-date-picker.component.tsx

```diff
@@ -11,14 +12,14 @@ const DEFAULT_PLACEHOLDER = 'dd/mm/yyyy';
 
 export interface OpenmrsDatePickerProps {
   id: string;
-  labelText: string;
+  labelText: string | any;
   onChange: (value: Date) => void;
   value?: Date | string;
   /* Not supported by Carbon's picker */
   defaultValue?: Date | string;
   minDate?: Date | string;
   maxDate?: Date | string;
-  readonly?: boolean;
+  readonly?: string | boolean;
```

Comment:
[Carbon only supports booleans here](https://github.com/carbon-design-system/carbon/blob/5078eb5204b21763244768af3b0b283551561d47/packages/react/src/components/DatePickerInput/DatePickerInput.tsx#L178). The RFE must convert these to booleans before passing the prop to the component. Otherwise, `"false"` will be treated as true.

```suggestion
  readonly?: boolean;
```